camel-core/src/main/java/org/apache/camel/ComponentVerifier.java

```diff
@@ -412,6 +411,12 @@ public String toString() {
         }
     }
 
+    class ExceptionAttributeImpl extends ErrorAttribute implements VerificationError.ExceptionAttribute {
+        private ExceptionAttributeImpl(String name) {
+            super(name);
+        }
+    }
+
```

Comment:
Ah, just saw that we can't move to o.a.c.impl.verifier since the access must be package scoped.